docker/prepare_proxy.sh

```diff
@@ -12,12 +12,14 @@ usage() {
   echo '  -u: Specify the UID of the user for which the redirection is not'
   echo '      applied. Typically, this is the UID of the proxy container'
   echo '  -i: Comma separated list of IP ranges in CIDR form to redirect to envoy (optional)'
+  echo '  -e: Comma separated list of IP ranges in CIDR form to exclude from outbound redirection to envoy (optional)'
   echo ''
 }
 
 IP_RANGES_INCLUDE=""
+IP_RANGES_EXCLUDE=""
```

Comment:
Please follow the style used in the new script:
 IP_RANGES_EXCLUDE=${IP_RANGES_EXCLUDE:-}
( i.e. allow it to be set as an env variable ).

We'll merge this into the new script in 0.3.